### PR TITLE
feat: Resolve empty clipart screen issue

### LINKED ISSUE #588: No message when saved clipart list is empty
**Describe the bug**
There is a little issue in the UI of the app. If there are no items in the saved cliparts then a blank screen appears to the user which is confusing and gives a bad experience.

**To Reproduce**
Delete all the saved cliparts, if any. Go to saved clipart fragment from the nav drawer.

**Expected behavior**
User should be informed that there is no saved clipart.

**Screenshots**

ezgif-2-870f847b6267

**Smartphone Info:**

|               |   |
|---------------|---|
|Device         | Mi Note 5 |
|Android Version|  Oreo 8.1  |

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by tronku**
Fixes #588 

Changes: Updated the UI as same as the empty badge layout.

Screenshots for the change:
WhatsApp Image 2020-01-02 at 9 47 09 PM

### CODE REVIEW

**Comment by adityastic** [comment]
@tronku can you use a nice placeholder, this might want to replaced with something good

**Comment by tronku** [comment]
> @tronku can you use a nice placeholder, this might want to replaced with something good

Okayy

**Comment by tronku** [comment]
Current placeholder
WhatsApp Image 2020-01-02 at 10 48 32 PM

**Comment by tronku** [comment]
@adityastic I have used gradle spotlessApply but it didn't change anything in the code format.

**Comment by adityastic** [comment]
@tronku use something that looks more pixelated

**Comment by adityastic** [comment]
> @adityastic I have used gradle spotlessApply but it didn't change anything in the code format.

We'll look at it after the main objective is finished, don't worry

**Comment by tronku** [comment]
New placeholder, more pixelated one

saved_clips

**Comment by adityastic** [comment]
@tronku looks good, please rebase to the latest changes

**Comment by adityastic** [comment]
also use the same icon for saved badges as well

**Comment by tronku** [comment]
Should I remove PreviewBadge view in the saved badge section if there are no badges in the list? Currently it's visible and doesn't make any sense.

**Comment by adityastic** [comment]
@tronku yes sure, that would be amazing 👍

**Comment by tronku** [comment]
Changed UI for both clipart and badge fragment and in landscape mode too.

**Comment by adityastic** [comment]
@tronku rebase

**Comment by tronku** [comment]
gradle spotlessApply is not working, not getting any issue in SavedClipartFragment.

File: app/src/main/res/drawable/ic_empty_list.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (adityastic) [inline app/src/main/res/drawable/ic_empty_list.xml]: why is this file still in app/

File: android/src/main/java/org/fossasia/badgemagic/ui/fragments/SavedBadgesFragment.kt [hunk 2]
```diff
@@ -1,188 +1,190 @@
-package org.fossasia.badgemagic.ui.fragments
```
Comment (adityastic) [inline android/src/main/java/org/fossasia/badgemagic/ui/fragments/SavedBadgesFragment.kt]: there are some issue with your changes. rebase your branch to the current development and then make these changes manually, then force push

**Comment by tronku** [comment]
After rebasing, I am not able to build the project. I think something went wrong due to multiplatform in my local project. I have resolved every conflict during rebase.

**Comment by adityastic** [comment]
> After rebasing, I am not able to build the project. I think something went wrong due to multiplatform in my local project. I have resolved every conflict during rebase.

Rebase interactively, drop all commits and remake those changes manually

**Comment by tronku** [comment]
After rebasing, there are lots of java files that get modified, multiplatform has now been integrated into my local project. Should I add all of them to the commit? There are 11 files

**Comment by adityastic** [comment]
@tronku there shouldn't be any extra files. All the required files are already added by me

**Comment by tronku** [comment]
@adityastic gradle spotlessApply is not working here.

**Pull request closed without merging**